kogitoq/kogito-common/src/main/java/life/genny/kogito/common/utils/KogitoUtils.java

```diff
@@ -161,6 +161,11 @@ public String sendSignal(final UseService useService, final String workflowId, f
 		log.info("Sending Signal to uri: " + uri);
 
 		HttpResponse<String> response = HttpUtils.post(uri, payload, "application/json", userToken);
+		if(response.statusCode() % 100 != 2) {
```

Comment:
Checking statusCode % 100 != 2 ensures that anything that isn't a result of success family is reported, but the success family is skipped (because it is successful)

https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/Response.Status.Family.html#SUCCESSFUL
We could use this but this does exactly the same thing
Is there any particular reason you want to check for specifically 200, vs the family of 200?